examples/hello-world-aks/README.md

```diff
@@ -0,0 +1,38 @@
+# Hello World AKS Example
+
+# Azure Kubernetes Service Confidential Hello World
+
+This sample is a basic Python application used to demonstrate Confidential Pods on Azure Kubernetes Service. In this sample an AMD SEV SNP report containing the container's firmware measurements will be displayed on the web using nginx.
+
+![Hello World Hardware Report](./media/hello-world-cc.png)
+
+The container is hosted publicly on [Azure Container Registry](fishersnpregistry.azurecr.io/hello-world-aks:1.0).
+
+## Getting Started
+
+To build the container run `./build.sh` from the `/examples/hello-world` directory.
+
+Then run `docker build -t <image-name:tag> .` from the same directory.
+
+## Deploying to Azure Kubernetes Service

```

Comment:
should we add a step for creating the RuntimeClass?

Reply:
Since it's going to be automatically injected by AKS, we can just fill the correct runtime name into the helloworld.yaml file once we know what it is.